### PR TITLE
Update dialog to use theme

### LINKED ISSUE #189: Background color of dialogs should match theme
When dialogs are shown to the user, they are always the default, white. I suggest that they should be updated to be consistent with the current theme. The white is jarring especially when the theme is a dark theme and the time of day is night.

### PR DESCRIPTION
**Pull request by peakvalleytech**
## :scroll: Description

Fixes #189
## :green_heart: How did you test it?

Verifed changes on multiple android versions.
## :camera_flash: Screenshots / GIFs

Screenshot_1616612404
Screenshot_1616612422
Screenshot_1616612390

### CODE REVIEW

File: app/src/main/java/journal/gratitude/com/gratitudejournal/ui/settings/SettingsFragment.kt [hunk 1]
```diff
@@ -138,6 +136,9 @@ class SettingsFragment : PreferenceFragmentCompat(),
             openShareApp()
             true
         }
+
+//        val firstDayOfWeek = findPreference<CustomListPreference>("first_day_of_week")
```
Comment (alisonthemonster) [inline app/src/main/java/journal/gratitude/com/gratitudejournal/ui/settings/SettingsFragment.kt]: Can you remove commented out code?

File: app/src/main/java/journal/gratitude/com/gratitudejournal/ui/settings/SettingsFragment.kt [hunk 2]
```diff
@@ -368,7 +368,12 @@ class SettingsFragment : PreferenceFragmentCompat(),
             dialogFragment.setTargetFragment(this, 0)
             dialogFragment.show(parentFragmentManager, "DIALOG")
         } else {
-            super.onDisplayPreferenceDialog(preference)
+            //super.onDisplayPreferenceDialog(preference)
```
Comment (alisonthemonster) [inline app/src/main/java/journal/gratitude/com/gratitudejournal/ui/settings/SettingsFragment.kt]: More commented out code to remove

File: app/src/main/res/layout/title_dialog.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/alertTitle"
+        style="@android:style/TextAppearance.Material.DialogWindowTitle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="TextView" />
+</LinearLayout>
```
Comment (alisonthemonster) [inline app/src/main/res/layout/title_dialog.xml]: Maybe I'm missing it, but where is this used?

File: ui/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -15,6 +15,18 @@
...
+        <item name="buttonBarNegativeButtonStyle">@style/NegativeButtonStyle</item>
+        <item name="buttonBarPositiveButtonStyle">@style/PositiveButtonStyle</item>
+        <!-- CHANGES THE DIALOG BACKGROUND COLOR!       -->
+        <item name="android:colorBackground">#555</item>
+        <!-- CHANGES BODY TEXT COLOR IN DIALOGS BUT NOT IN TIME PICKER     -->
+        <item name="android:textColorPrimary">#363636</item>
+        <!-- CHANGES HEADER TEXT COLOR IN DIALOGS BUT NOT IN TIME PICKER     -->
+        <item name="android:textColor">#000000</item>
+    </style>
+
```
Comment (alisonthemonster) [inline ui/src/main/res/values/styles.xml:17]: Why not modify the `AlertDialogTheme` above this one? Where is this being used?

Shouldn't we also update these colors to use the attributes?

Comment (peakvalleytech) [reply ui/src/main/res/values/styles.xml:17]: @alisonthemonster
Ok, i've  sent a new commit.

**Comment by peakvalleytech** [comment]
@alisonthemonster  Ok, just sent a new commit.  Also, I wanted to say that I love the new icon. It's great.

**Comment by alisonthemonster** [comment]
Running into issues with CI today, the red x is on my end and not yours. Will run again when the issue is resolved!

**Pull request merged**